src/sparse_variational.jl

```diff
@@ -142,12 +298,19 @@ function AbstractGPs.elbo(
     ::SparseVariationalApproximation, ::FiniteGP, ::AbstractVector; kwargs...
 )
     return error(
-        "The observation noise fx.Σy must be homoscedastic.\n To avoid this error, construct fx using: f = GP(kernel); fx = f(x, σ²)",
+        "The observation noise fx.Σy must be homoscedastic.\n To avoid this error, " *
+        "construct fx using: f = GP(kernel); fx = f(x, σ²)",
```

Comment:
```suggestion
        "The observation noise fx.Σy must be homoscedastic.\n To avoid this error, ",
        "construct fx using: f = GP(kernel); fx = f(x, σ²)",
```
`error` can take multiple arguments

Reply:
Oooooo